### PR TITLE
Address PR comments

### PR DESCRIPTION
**Pull request by jeffreylovitz**
Resolves comments made on #1904

### CODE REVIEW

File: tests/flow/test_config.py [hunk 1]
```diff
@@ -168,5 +176,41 @@ def test08_config_restore_timeout_to_default(self):
...
+        # non-negative values, 0 resets to default
+        for config in ["TIMEOUT", "QUERY_MEM_CAPACITY", "DELTA_MAX_PENDING_CHANGES"]:
+            try:
+                redis_con.execute_command("GRAPH.CONFIG SET %s -1" % config)
+                assert(False)
+            except redis.exceptions.ResponseError as e:
+                assert("Failed to set config value %s to -1" % config in str(e))
+                pass
+
+        # RESULTSET_SIZE can be any integer, negative values reset to default
```
Comment (swilly22) [inline tests/flow/test_config.py:210]: Can you please point me to where do we treat negative values as reverting to default in this case ?

Comment (jeffreylovitz) [reply tests/flow/test_config.py:210]: https://github.com/RedisGraph/RedisGraph/blob/master/src/configuration/config.c#L216

Comment (swilly22) [reply tests/flow/test_config.py:210]: But resultset_size is defined as `uint64_t resultset_size` seems like it would never be negative

**Review by swilly22 — CHANGES_REQUESTED** [review]

File: tests/flow/test_config.py (hunk 1 above)
Comment (jeffreylovitz) [reply tests/flow/test_config.py:210]: The config, like all others, are read from Redis as a `long long`. It's then cast to an `int64_t` when calling `Config_resultset_max_size_set`, then set as a `uint64_t` when saved in the struct - `UINT64_MAX` if negative.

Do you want me to make 0 reset to the default to make this more consistent with the others?

Comment (swilly22) [reply tests/flow/test_config.py:210]: No that's fine, 0 is a valid result-set size, although I don't think anyone will use it, also as you've said this diverts from most other configurations.

Comment (swilly22) [inline tests/flow/test_config.py:210]: `# RESULTSET_SIZE can be any integer, negative values reset to default` is this tested?

File: tests/flow/test_config.py [hunk 2]
```diff
@@ -168,5 +176,41 @@ def test08_config_restore_timeout_to_default(self):
...
+        try:
+            redis_con.execute_command("GRAPH.CONFIG SET RESULTSET_SIZE invalid")
```
Comment (swilly22) [inline tests/flow/test_config.py:212]: can we extend this as above to be invoked on all other types of configurations ?

**Review by swilly22 — APPROVED** [review]

**Pull request merged**